spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/ErrorHandling.scala

```diff
@@ -36,4 +37,8 @@ trait ErrorHandling {
       .getOrElse(ErrorMessageSubmitWithoutColumn(errType, errCode, errMessage, additionalInfo))
     putErrorToColumn(toSubmit)
   }
+
+  def errorColumnType: DataType
+
+  def errorColumnAggregationType: Option[DataType]
```

Comment:
David and I talked (in one of his previous PRs I think) that it can be nice to put abstract methods/functions into upper parts of a given class/trait. Would you mind just moving it to the top / more up?

Reply:
I will try to.

Reply:
yes, the most 'abstract' one. But I don't know if you agree or disagree with my comment...?

Reply:
I agree with your comment.

Reply:
Still there ^, just remaining